ssl/record/methods/tls_common.c

```diff
@@ -1916,8 +1916,11 @@ int tls_retry_write_records(OSSL_RECORD_LAYER *rl)
             } else {
                 if (BIO_should_retry(rl->bio))
                     ret = OSSL_RECORD_RETURN_RETRY;
-                else
+                else {
+                    ERR_raise_data(ERR_LIB_SSL, get_last_sys_error(),
```

Comment:
typo 
```suggestion
                    ERR_raise_data(ERR_LIB_SYS, get_last_sys_error(),
```

Reply:
No typo, the intent was to raise this error against the OpenSSL lib specifically, to keep it in line with the sendfile variant

Reply:
I believe it has been changed to `ERR_LIB_SYS` in this very pull request.
https://github.com/openssl/openssl/blob/93f4a92c558521cc82abb289640819cc7e6dbeb4/ssl/ssl_lib.c#L2602

Reply:
crud, you're right, I didn't reconcile them properly, will take care of this